src/StateService/StateService/config.json

```diff
@@ -1,6 +1,12 @@
 {
   "PluginConfiguration": {
     "Path": "Data_MPT_{0}",
-    "FullState": false
+    "FullState": false,
+    "VerifierUrls": [
+      "http://state1.ngd.network:10332",
+      "http://state2.ngd.network:10332",
+      "http://state3.ngd.network:10332",
+      "http://state4.ngd.network:10332"
```

Comment:
Maybe we should set empty here @ZhangTao1596 @vncoelho

Reply:
Maybe @Tommo-L, I still need to understand some aspects. 
Thanks for the previous explanations.

If you have some time to take a look at my last comment on our previous thread.

Reply:
Done.